.env.local.example

```
@@ -39,3 +39,4 @@ ETHERSCAN_API_KEY=
 # Monarch monitoring API for trending markets
 MONARCH_API_ENDPOINT=http://localhost:3000
 MONARCH_API_KEY=
+BROWSE_BLACKLIST_GIST_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BROWSE_BLACKLIST_GIST_URL key should go before the MONARCH_API_ENDPOINT key

(UnorderedKey)